MekHQ/src/mekhq/campaign/unit/Unit.java

```diff
@@ -4093,10 +4093,12 @@ public void addDriver(Person p, boolean useTransfers) {
     }
 
     public void addGunner(Person p) {
-        addGunner(p, false);
+        addGunner(Objects.requireNonNull(p), false);
```

Comment:
Doesn't this duplicate the nonNull check?

Reply:
Personal preference for error call stacks, but I'll simplify.